example/src/main/res/animator/animator_custom_tick.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<objectAnimator xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="600"
+    android:propertyName="rotation"
+    android:repeatMode="restart"
+    android:repeatCount="infinite"
```

Comment:
I would not add an animator with a `repeatCount="infinite"` as it's sort of confusing for the user. The feature is that we support adding another type of Icon, not that we allow them to loop. So maybe another easier animation would also work.